flow/utils/rllib.py

```diff
@@ -207,6 +208,9 @@ def get_rllib_config(path):
 
 def get_rllib_pkl(path):
     """Return the data from the specified rllib configuration file."""
+    dirname = os.path.dirname(__file__)
```

Comment:
This is risky, this breaks if you run it from a different folder. You want os.getcwd I think

Reply:
Done